analyses/molecular-subtyping-ATRT/00-subset-files-for-ATRT.R

```diff
@@ -142,8 +147,26 @@ readr::write_tsv(filtered_gsva_scores, file.path(results_dir, "atrt_gsva.tsv"))
 tmb_df <- tmb_df %>%
   dplyr::select(Tumor_Sample_Barcode, tmb) %>%
   dplyr::left_join(select_metadata,
-                    by = c("Tumor_Sample_Barcode" = "Kids_First_Biospecimen_ID")) %>%
+                   by = c("Tumor_Sample_Barcode" = "Kids_First_Biospecimen_ID")) %>%
   dplyr::filter(sample_id %in% atrt_df$sample_id)
 
 # Write to file
 readr::write_tsv(tmb_df, file.path(results_dir, "atrt_tmb.tsv"))
+
+#### Filter GISTIC data --------------------------------------------------------
+
+gistic_df <- gistic_df %>%
+  as.data.frame() %>%
```

Comment:
Do you need to call `as.data.frame()` here to get the `tibble::column_to_rownames` step to work? Asking because you set this as a `data.frame` after the transpose. If it's necessary to do this twice, I think it may be because `gistic_df` is a `data.table` (see my comment above).

Reply:
Yes, you are correct.